Dockerfile

```diff
@@ -1,31 +1,29 @@
-FROM debian:jessie
-MAINTAINER Luke Campbell <luke.campbell@rpsgroup.com>
+FROM node:13.7.0-alpine
+MAINTAINER RPS Dev-Ops <devops@rpsgroup.com>
 
-ENV NODE_VERSION 7.9.0
-ENV GOSU_VERSION 1.9
-ENV SCRIPTS_DIR /opt/build_scripts
-
-RUN mkdir -p $SCRIPTS_DIR
-RUN useradd -m node
-
-COPY contrib/scripts/ $SCRIPTS_DIR/
-
-RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
-
-RUN $SCRIPTS_DIR/install-deps.sh
-RUN $SCRIPTS_DIR/install-node.sh
 COPY bin /opt/ioos-us/bin
```

Comment:
Since this are going to the same destination parent directory, you could you the multi-argument form of `COPY` here.